pkgs/development/compilers/llvm/10/clang/default.nix

```diff
@@ -90,6 +90,8 @@ let
     passthru = {
       inherit libllvm;
       isClang = true;
+      defaultCStandard = 11;
```

Comment:
Can this be treated as int?

Reply:
I'm not sure what you mean?
In case you are thinking of `gnu11` vs `c++11`: That almost never makes a difference. Allowing arithmetic on this value is what makes it useful.

Reply:
Understandable.
The problem to me is because e.g C20 comes after C99 however 20<99.

Reply:
This is another reason why it's better to detect "is the compiler going to use a standard equal to or later than XYZ" like autoconf does -- no need to encode the entire list of standards in nix, along with their partial order.